### DISCUSSION TITLE
New Quasar Extension

### DISCUSSION THREAD

**Discussion by odranoelBR** [quasarframework/quasar#9184] [Quasar App Extensions]
Hi, community!

I created a QSelect with automatic fetch/populate from an API, and easy filter.
I hope it helps with some projects.

https://github.com/odranoelBR/vue-quasar-select-api

**Comment by donPuerto** [comment]
Nice, Will this work on latest version of Quasar (Vue 3)?

**Reply by odranoelBR** [reply]
Thankyou @donPuerto 

Yes, vue 3 quasar 2 ROCKS!
And i use this extension in some projects.
